src/hooks/useMovieDetails.js

```diff
@@ -5,8 +5,18 @@ export default function useMovieDetails(id) {
   const [movieCast, setMovieCast] = useState([]);
   const [movieWatchProviders, setMovieWatchProviders] = useState([]);
   const [similarMovies, setSimilarMovies] = useState([]);
+  const [movieTrailer, setMovieTrailer] = useState([]);
   const [isLoading, setIsLoading] = useState(false);
 
+  const movieTrailerUrl = movieTrailer
+    ?.filter(
+      video =>
+        video.site === 'YouTube' &&
+        (video.type === 'Trailer' || video.type === 'Teaser') &&
+        video.size >= 720
+    )
+    ?.map(video => `https://www.youtube.com/watch?v=${video.key}wowK7ADGRsQ`);
```

Comment:
I think the use of the optional chaining is a bit too heavy/too much 😅.

Reply:
Oh you're right, that was one more holdover to find a bug